testing/internal/e2e/boundary/target.go

```diff
@@ -61,14 +61,58 @@ func CreateNewTargetCli(t testing.TB, ctx context.Context, projectId string, def
 	return newTargetId
 }
 
+// CreateNewAddressTargetCli uses the cli to create a new target using an
+// address in boundary.
+// Returns the id of the new target.
+func CreateNewAddressTargetCli(t testing.TB, ctx context.Context, projectId string, defaultPort string, address string) string {
+	output := e2e.RunCommand(ctx, "boundary",
+		e2e.WithArgs(
+			"targets", "create", "tcp",
+			"-scope-id", projectId,
+			"-default-port", defaultPort,
+			"-name", "e2e Target",
+			"-address", address,
+			"-format", "json",
+		),
+	)
+	require.NoError(t, output.Err, string(output.Stderr))
+	var newTargetResult targets.TargetCreateResult
+	err := json.Unmarshal(output.Stdout, &newTargetResult)
+	require.NoError(t, err)
+	newTargetId := newTargetResult.Item.Id
+	t.Logf("Created Target: %s", newTargetId)
+
+	return newTargetId
+}
+
 // AddHostSourceToTargetCli uses the cli to add a host source (host set or host) to a target
-func AddHostSourceToTargetCli(t testing.TB, ctx context.Context, targetId string, hostSourceId string) {
+func AddHostSourceToTargetCli(t testing.TB, ctx context.Context, targetId, hostSourceId string) {
```

Comment:
Is this a general pattern that is used? Does this mean that `targetId` and `hostSourceId` are strings?

Reply:
Yea, If you have a function signature with multiple arguments of the same type, you don't have to repeat yourself and write the type of every argument.